src/plugins/home/public/application/components/tutorial/tutorial.js

```diff
@@ -255,6 +253,9 @@ class TutorialUi extends React.Component {
           <EuiFlexItem grow={false}>
             <EuiButtonGroup
               options={radioButtons}
```

Comment:
bump :)

Reply:
Okay seems like this actually isn't necessary at the moment since the Tutorials were removed in https://github.com/opensearch-project/OpenSearch-Dashboards/pull/464